build_docs.pl

```diff
@@ -539,14 +537,8 @@ sub push_changes {
             run qw(git push origin HEAD );
         }
         local $ENV{GIT_DIR} = $target_repo->git_dir if $target_repo;
-        if ( $Opts->{staging} ) {
-            say "Force pushing changes to staging";
-            run qw(git push -f origin HEAD );
```

Comment:
This `-` diff right here is the code equivalent of warding off the Evil Eye hahaha

Reply:
One of the PRs you approved two days ago also had push -f!

Reply:
😐